### ISSUE TITLE
[Bug][a11y] Edit bookmarks - bookmarks folder needs resizing

### ISSUE THREAD

**Issue by sflorean** [mozilla-mobile/fenix#17903]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

## Steps to reproduce
1. Add a bookmark.
2. Tap on "Edit" from the confirmation snakbar.
3. Using the Accessibility Scanner, scan the "Edit bookmark section".

### Expected behavior
The "Edit bookmark" section is properly scanned and no suggestions are made. 

### Actual behavior
![Screenshot_20210208_110826_com google android apps accessibility auditor](https://user-images.githubusercontent.com/58466444/107210435-cc9e2480-6a0c-11eb-9fdb-1fcc3b731c62.jpg)

### Device information
* Device vendor / model and Android version: Huawei P40 Lite (Android 11)
* Firefox for Android version: 86.0.0-beta.2

**Comment by neha-b2001** [comment]
Hello, @eliserichards. I am an Outreachy applicant.

**Comment by neha-b2001** [comment]
Thanks for the assignment 😄

**Comment by neha-b2001** [comment]
@eliserichards
Actually, in `fragment_edit_bookmark.xml`, `bookmarkParentFolderSelector` has already been set to 48dp height: 

![image](https://user-images.githubusercontent.com/67039214/113287661-7e6e1900-930b-11eb-8d73-59fdcbe0a314.png)

However, Google Accessibility Scanner is giving the **Multiple items have same description** suggestion to all three elements of this fragment:

![image](https://user-images.githubusercontent.com/67039214/113288220-49ae9180-930c-11eb-80ef-599f07e82cc7.png)

Should I change the description of these Views then? (For eg, BOOKMARK URL instead of URL or something like that)

**Comment by eliserichards** [comment]
I think that can be broken out into its own story @neha-b2001. I'll close this issue since it is no longer valid.

**Comment by neha-b2001** [comment]
> I think that can be broken out into its own story @neha-b2001. I'll close this issue since it is no longer valid.

Okay, thanks! 👍
